src/workflows/bidsRoiBasedGLM.m

```diff
@@ -0,0 +1,115 @@
+% (C) Copyright 2021 CPP BIDS SPM-pipeline developers
+
+function bidsRoiBasedGLM(opt)
```

Comment:
this function is for running GLM on only ROIs, right? instead of whole brain. Here is a irrelevant question! 
I usually do not prefer roi-based GLM, and go with whole brain. Then extract the betas from a given ROI. Do you know which functions could be useful for extracting betas in a given ROI in cpp-spm?

Reply:
`spm_summarize` ?